v_5_0_0/files/conf/k8s-addons

```diff
@@ -141,6 +141,19 @@ do
     sleep 5s
 done
 
+# apply network policies:
+NETWORK_POLICIES_FILE="network_policies.yaml"
+NAMESPACES="default giantswarm"
```

Comment:
This should be kube-system

Reply:
yeah noted that as I wrote above:

> oh noted that I've put there default rather than kube-system

;-)